frontend/frontend_test.go

```diff
@@ -239,6 +241,7 @@ func testRefReadDir(t *testing.T, sb integration.Sandbox) {
 }
 
 func testRefStatFile(t *testing.T, sb integration.Sandbox) {
+	integration.SkipOnPlatform(t, "windows")
```

Comment:
I think this might pass.

Reply:
Same to this, was failing in my previous runs but I can look into this too as the first batch for the next PR.